docs/quickstarts/nextjs.mdx

```diff
@@ -3,15 +3,39 @@ title: Use Clerk with Next.js
 description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your Next.js application. Clerk works seamlessly on both client side and server side components.
 ---
 
-# Use Clerk with Next.js
+# Add authentication and user management to your Next.js app with Clerk
 
-Learn how to use Clerk to quickly and easily add secure authentication and user management to your Next.js application. Clerk works seamlessly in both client side and server side components.
+## You will learn how to
+
+* Install `@clerk/nextjs`
+* Set up your environment keys
+* Wrap your Next.js app in `<ClerkProvider />`
+* Limit access to authenticated users
+* Embed the `<UserButton />`
+
+**Estimated time** 1 cup of coffee
+
+### Prerequisites 

```

Comment:
We can add a little more context to these bullet points. So for example, our social connection pages have this on every page:
## Before you start
- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](https://clerk.com/docs/quickstarts/setup-clerk).
- You need to have a <Insert Social connection> developer account. To create one, visit the [<Insert Social connection> console](#).

E.g. see [Google social connection](https://clerk.com/docs/authentication/social-connections/google) page.